.coveragerc

```diff
@@ -1,4 +1,9 @@
 [run]
 parallel = True
 data_file = /tmp/${USER}/cov/coverage.db
-omit = */site-packages/*
+source =
+    lib/pbench
+omit =
+    */site-packages/*
+    /tmp/${USER}/tox/*/bin/bottle.py
+    /tmp/${USER}/tox/*/lib/python3*
```

Comment:
What else do you see here? I don't have an agent tox environment fired up, but `ls /tmp/dbutenho/tox/*/lib/python3*/` only shows me `site-packages`, which is already omitted above.

Reply:
I had another part of the tree with a `distutils` hierarchy.

Reply:
I've added comments here to record why we need these omissions.

Reply:
When the `source` directive is not used, we see `distutils` as well as `site-packages`.  When using `source` we don't see anything now.